pages/journal.js

```diff
@@ -3,16 +3,320 @@ import Helmet from 'react-helmet';
+			  }, {
+				  author: "Maya Agarwal",
+				  title: "Two Versions of the Ramayana and the Concept of the Lokasamgraha"
+			  }, {
+				  author: "Pisacha Wichianchan",
+				  title: "The Damned Politics of Sarawak: Rethinking the Bakun Megaproject in Malaysia"
+			  }, {
+				  author: "Hoo Ri Kim",
+				  title: "An Assessment of the Conflict Management Tools of the Korean Conflict"
+			  }, {
+				  author: "Yuma Kuwata",
+				  title: "Policy Recommendation for the U.S. Government in Reaction to Japan's Possible Nuclear Energy Policy Reformation"
+			  }
+		  ]
+      }, {
+		title: 'BSJAS Third Edition',
+        year: 'Spring 2013',
+		cover: '/assets/2013cover.png',
+        content: [
+			  {
+				  author: "Kankan Xie",
+				  title: "How Does Ethnicity Affect Current China-Malaysia Relations?"
+			  }, {
+				  author: "Mark Portillo",
+				  title: "Constructing \"Culture\""
+			  }, {
+				  author: "Rachel Phoa",
+				  title: "The Success of Environmental NGOs in Suharto's Indonesia"
+			  }, {
+				  author: "Ramanathan Veerappan",
+				  title: "The Relevance of Kautilya's Arthashastra"
+			  }, {
+				  author: "Valerie Black",
+				  title: "Ainu and the Metaphors of Indigeneity: Land, History, Science, Law and Bodies"
+			  }, {
+				  author: "Robert Bowers Curl",
+				  title: "Not so Singular: Laicization in Japanese Buddhism and Parallels in South Asia"
+			  }
+		  ]
+      }, {
+		title: 'BSJAS Second Edition',
+        year: 'Spring 2012',
+		cover: '/assets/2012cover.png',
+        content: [
+			  {
+				  author: "Dawei Ding",
+				  title: "A Sociopolitical and Economic Analysis of Recent Shanghai Policies on Hukou and Migrant Worker Insurance"
+			  }, {
+				  author: "Tara Yarlagadda",
+				  title: "An Analysis of State-based Narratives in Pakistani and Indian Textbooks"
+			  }, {
+				  author: "Muyang Chen",
+				  title: "The Current Buraku Issue in Tokyo and Its Solutions"
+			  }, {
+				  author: "Benjamin Knight",
+				  title: "Deconstructing Japan's Military Globalism: The March to \"Normal\" Nationhood"
+			  }
+		  ]
+      }, {
+		title: 'BSJAS Inaugural Edition',
+        year: 'Fall 2010',
+		cover: '/assets/2010cover.png',
+        content: [
+			  {
+				  author: "James Lin",
+				  title: "Soviet Internationalism, Chinese Nationalism, and Early Kuomintang Struggles"
+			  }, {
+				  author: "Alessandro Tiberio",
+				  title: "Becoming Taiwanese"
+			  }, {
+				  author: "Andrea Horbinski",
+				  title: "The Tokyo Tribunal"
+			  }, {
+				  author: "Nina Tompkin",
+				  title: "Professor Interview featuring Lanchih Po"
+			  }
+		  ]
+      },
+	  
+    ];
+
+    const handleClick = (index) => {
+      this.setState({
+        activeTab: index,
+      });
+    }
+
+    const tabs = () => {
+      let tabs = [{title: 'Spring 2015'}, {title: 'Spring 2014'}, {title: 'Spring 2013'}, {title: 'Spring 2012'}, {title: 'Fall 2010'}];
+      const HTMLify = function (item, i) {
```

Comment:
look up the es6 notation for map and try to use that instead, makes things much simpler in terms of code structure